modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -87,9 +87,9 @@ Note the worker with the role `worker-rt` that contains the string `4.18.0-211.r
 NAME                               	STATUS   ROLES           	AGE 	VERSION                  	INTERNAL-IP
 EXTERNAL-IP   OS-IMAGE                                       	KERNEL-VERSION
 CONTAINER-RUNTIME
-cnf-worker-0.example.com	          Ready	 worker,worker-rt   5d17h   v1.22.0
+cnf-worker-0.example.com	          Ready	 worker,worker-rt   5d17h   v1.22.1
 128.66.135.107   <none>    	        Red Hat Enterprise Linux CoreOS 46.82.202008252340-0 (Ootpa)
-4.18.0-211.rt5.23.el8.x86_64   cri-o://1.22.0-90.rhaos4.8.git4a0ac05.el8-rc.1
+4.18.0-211.rt5.23.el8.x86_64   cri-o://1.22.1-90.rhaos4.8.git4a0ac05.el8-rc.1
```

Comment:
Prolly doesn't matter, but my expectation is that the other values on this line change too.

Reply:
Yeah possibly, but it's just example output, so we do the best we can. And without rerunning them all, we probably won't know what else to update.